eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheet.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import TableContainer from "@material-ui/core/TableContainer";
+import Paper from "@material-ui/core/Paper";
+import { DateTime } from "luxon";
+import { useTranslation } from "react-i18next";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+export interface Props {
+  date: DateTime;
+}
+
+const AttendanceSheet: React.FC<Props> = ({ date, children }) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  return (
+    <>
+      <div className={classes.title}>{t("AttendanceSheet.Date", { date })}</div>
```

Comment:
Just a suggestion: I would maybe split the title and the date in two elements (`h3` and `span`) and put the date to the right edge of the row, without the parentheses (and possibly italic)